docs/usage/channels.rst

```diff
@@ -0,0 +1,452 @@
+   manager
+
+Both :meth:`subscribe <ChannelsPlugin.subscribe>` and
+:meth:`start_subscription <ChannelsPlugin.start_subscription>` produce a
+:class:`Subscriber`, which can be used to interact with the streams of events subscribed
+to.
+
+The context manager should be preferred, and using the methods directly should only be
+done when a context manager cannot be used, e.g. when the subscription would span
+different contexts.
+
+
+.. code-block:: python
+    :caption: Calling the subscription methods manually
+
+    subscriber = await channels.subscribe(["foo", "bar"])
+    try:
+        ...  # do some stuff here
+    finally:
+        await channels.unsubscribe(subscriber)
+
+
+
+.. code-block:: python
+    :caption: Using the context manager
+
+    async with channels.start_subscription(["foo", "bar"]) as subscriber:
+        ...  # do some stuff here
+
+
+It is also possible to unsubscribe from individual channels, which may be desirable if
+subscriptions need to be managed dynamically.
+
+.. code-block:: python
+
+    subscriber = await channels.subscribe(["foo", "bar"])
+    ...  # do some stuff here
+    await channels.unsubscribe(subscriber, ["foo"])
+
+
+Or, using the context manager
+
+.. code-block:: python
+
+    async with channels.start_subscription(["foo", "bar"]) as subscriber:
+        ...  # do some stuff here
+        await channels.unsubscribe(subscriber, ["foo"])
+
+
+
+Managing history
++++++++++++++++++
+
+Some backends support a per-channel history, keeping a certain amount of
+:term:`events <event>` in storage. This history can then be pushed to a
+:term:`subscriber`.
+
+The plugin's :meth:`put_subscriber_history <ChannelsPlugin.put_subscriber_history>` can
+be used to fetch this history and put it into a subscriber's :term:`event stream`.
+
+.. literalinclude:: /examples/channels/put_history.py
+    :language: python
+
+
+.. note::
+    The publication of the history happens sequentially, one channel at a time, one
+    event at a time. This is done to ensure correct ordering of the events and to not
+    fill up a subscriber's backlog, resulting in dropped history entries. Should the
+    amount of entries exceed the maximum backlog size, the execution will wait until
+    previous events have been processed.
+
+    .. seealso::
+        `Managing backpressure`
+
+
+The ``Subscriber``
+------------------
+
+.. py:currentmodule:: litestar.channels.subscriber
+
+The :class:`Subscriber` manages an individual :term:`event stream`, provided to it by
+the plugin, representing the sum of events from all channels the subscriber has
+subscribed to.
+
+It can be considered the endpoint of all events, while the backends act as the source,
+and the plugin as a router, being responsible for supplying events gathered from the
+backend into the appropriate subscriber's streams.
+
+In addition to being an abstraction of an :term:`event stream`, the :class:`Subscriber`
+provides different methods to handle this stream:
```

Comment:
```suggestion
provides different methods to interact with it:
```

Reply:
See previous comment. There is only one method to interact with the stream, but different ways to handle it.